src/test/java/io/pravega/connectors/flink/FlinkPravegaWriterTest.java

```diff
@@ -150,7 +149,7 @@ private void runTest(final int jobParallelism, final boolean withFailure, final
                 setParallelism(jobParallelism);
         execEnv.setRestartStrategy(RestartStrategies.fixedDelayRestart(1, 0));
 
-        DataStreamSource<Integer> dataStream = execEnv.enableCheckpointing(1000, CheckpointingMode.EXACTLY_ONCE)
```

Comment:
Evidently the `FlinkPravegaWriterTest` was testing the non-transactional writer with snapshotting enabled, since it had some code (in `snapshotState`) for that mode.